vsintegration/src/FSharp.Editor/LanguageService.fs

```diff
@@ -69,14 +80,23 @@ type internal FSharpLanguageService(package : FSharpPackage) =
                     optionsCache.Add(projectId, options)
 
                 if obj.ReferenceEquals(workspace.ProjectTracker.GetProject(projectId), null) then
-                    let projectSite = new FSharpProjectSite(hier, this.SystemServiceProvider, workspace, projectFileName);
-                    projectSite.Initialize(hier, site)                    
+                    let projectContextFactory = this.Package.ComponentModel.GetService<IWorkspaceProjectContextFactory>();
+                    let errorReporter = ProjectExternalErrorReporter(projectId, "FS", this.SystemServiceProvider)
+                    let outputFlag = (site.CompilerFlags() |> Seq.find(fun flag -> flag.StartsWith("-o:"))).Substring(3)
```

Comment:
This is also legal:
--out:<file>                             Name of the output file (Short form: -o)